src/iOS.Core/Renderers/CollectionView/ExtendedGroupableItemsViewController.cs

```diff
@@ -38,7 +38,6 @@ protected override void UpdateTemplatedCell(TemplatedCell cell, NSIndexPath inde
                     // Do nothing in here, this is temporary to get more info about the crash, if the logger fails, we want to get the info
                     // by crashing with the original exception and not the logger one
                 }
-                throw colEx;
```

Comment:
🤔 Could we allow throwing all exceptions except the one that is being reported in AppCenter while crashing? If I'm not mistaken it would be `IndexOutOfRangeException`, so basically adding:
```c#
if (ex is IndexOutOfRangeException)
{
     return;
}
throw colEx;
```
Because if we don't throw anything we might lose information on AppCenter on non-related issues to this one.